### DISCUSSION TITLE
0220. 存在重复元素 III

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#148] [Comments]
# 0220. 存在重复元素 III

[https://algo.itcharge.cn/solutions/0200-0299/contains-duplicate-iii/](https://algo.itcharge.cn/solutions/0200-0299/contains-duplicate-iii/)

**Comment by ZKFcoder** [comment]
学不会 我废了

**Reply by HONG-ZI** [reply]
同废物......
